doc/md/privacy.md

```diff
@@ -421,26 +425,26 @@ func (TenantMixin) Policy() ent.Policy {
 
 Then, after running the code-generation, we expect the privacy-rules to take effect on the client operations.
 
-```go
+```go title="examples/privacytenant/example_test.go"
 func Do(ctx context.Context, client *ent.Client) error {
     // A continuation of the code-block above.
 
-	// Create 2 users connected to the 2 tenants we created above (a8m->GitHub, nati->GitLab).
-	a8m := client.User.Create().SetName("a8m").SetTenant(hub).SaveX(admin)
-	nati := client.User.Create().SetName("nati").SetTenant(lab).SaveX(admin)
+	// Create 2 users connected to the 2 tenants we created above
+	a8mHub := client.User.Create().SetName("a8mHub").SetTenant(hub).SaveX(adminCtx)
+	natiLab := client.User.Create().SetName("natiLab").SetTenant(lab).SetFoods([]string{"Sushi", "Burritos"}).SaveX(adminCtx)
```

Comment:
```suggestion
	hubUser := client.User.Create().SetName("a8m").SetTenant(hub).SaveX(adminCtx)
	labUser := client.User.Create().SetName("nati").SetTenant(lab).SaveX(adminCtx)
```

Please use these names instead if you find the previous example unclear. Also, the `foods` field is irrelevant for this example.